tt-metal-sdxl/config/constants.py

```diff
@@ -0,0 +1,100 @@
+from enum import Enum
+
+# MODEL environment variable
+class SupportedModels(Enum):
+    STABLE_DIFFUSION_XL_BASE = "stable-diffusion-xl-base-1.0"
+    STABLE_DIFFUSION_3_5_LARGE = "stable-diffusion-3.5-large"
+    DISTIL_WHISPER_LARGE_V3 = "distil-whisper/distil-large-v3"
+    MICROSOFT_RESNET_50 = "microsoft/resnet-50"
+
+class ModelRunners(Enum):
+    TT_SDXL = "tt-sdxl"
+    TT_SDXL_TRACE = "tt-sdxl-trace"
+    TT_SD3_5 = "tt-sd3.5"
+    TT_WHISPER = "tt-whisper"
+    FORGE = "forge"
+    MOCK = "mock"
+
+class ModelServices(Enum):
+    IMAGE = "image"
+    AUDIO = "audio"
+    TEXT = "text"
+
+# DEVICE engvironment variable
+class DeviceTypes(Enum):
+    N150 = "n150"
+    N300 = "n300"
+    GALAXY = "galaxy"
+    QUIETBOX = "quietbox"
+
+# Combined model-device specific configurations
+# useful when whole device is being used by a single model type
+# also for CI testing
+ModelConfigs = {
+    (SupportedModels.STABLE_DIFFUSION_XL_BASE, DeviceTypes.N150): {
+        "model_runner": "tt-sdxl-trace",
```

Comment:
You can use `ModelRunners.TT_SDXL_TRACE` here. You can apply the same pattern to other model configs.
```suggestion
        "model_runner": ModelRunners.TT_SDXL_TRACE,
```